src/gui/owncloudgui.cpp

```diff
@@ -699,8 +709,6 @@ void ownCloudGui::slotUpdateProgress(const QString &folder, const ProgressInfo&
             _recentItemsActions.takeFirst()->deleteLater();
         }
         _recentItemsActions.append(action);
-
-        slotRebuildRecentMenus();
```

Comment:
If this is removed, how is _recentActionsMenu now updated while the menu is shown during sync?